google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/mutaterows/MutateRowsAttemptCallable.java

```diff
@@ -268,6 +281,8 @@ private void handleAttemptSuccess(List<MutateRowsResponse> responses) {
     List<Integer> newOriginalIndexes = Lists.newArrayList();
     boolean[] seenIndices = new boolean[currentRequest.getEntriesCount()];
 
+    ErrorDetails errorDetails = null;
```

Comment:
I'm quite uncertain about embedding error details per entry. More so, I'm really weirded out about using the last entry-with-error-details to be the overall error details for the MutateRowsException.